<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ECH Workers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <!-- 工具栏 -->
    <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset theme="list-add"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>编辑</string>
        </property>
        <property name="icon">
         <iconset theme="document-edit"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset theme="list-remove"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTest">
        <property name="text">
         <string>测试</string>
        </property>
        <property name="icon">
         <iconset theme="network-wired"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestAll">
        <property name="text">
         <string>全部测试</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImport">
        <property name="text">
         <string>导入</string>
        </property>
        <property name="icon">
         <iconset theme="document-import"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="text">
         <string>导出</string>
        </property>
        <property name="icon">
         <iconset theme="document-export"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="toolbarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSystemProxy">
        <property name="text">
         <string>系统代理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkTunMode">
        <property name="text">
         <string>TUN 模式</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStartStop">
        <property name="text">
         <string>启动</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>QPushButton { font-weight: bold; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- 主内容区域 (分割器) -->
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <!-- 节点列表 -->
      <widget class="QTableWidget" name="nodeTable">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <!-- 日志区域 -->
      <widget class="QTextBrowser" name="logBrowser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>日志输出...</string>
       </property>
      </widget>
     </widget>
    </item>
    <!-- 状态栏 -->
    <item>
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>未运行</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="statusSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelVersion">
        <property name="text">
         <string>v1.0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
